project/KernelBoiler.scala

```diff
@@ -150,7 +150,9 @@ object KernelBoiler {
                 |      def empty: ${`(A..N)`} = ${nullaryTuple("empty")}
                 |      def inverse(x: ${`(A..N)`}): ${`(A..N)`} = ${unaryTuple("inverse")}
                 |    }
-                |  implicit def catsKernelStdOrderForTuple${arity}[${`A..N`}](implicit ${constraints("Order")}): Order[${`(A..N)`}] =
+                |  implicit def catsKernelStdOrderForTuple${arity}[${`A..N`}](implicit ${constraints(
+                  "Order"
+                )}): Order[${`(A..N)`}] =
```

Comment:
This is what I meant in #4101 – not sure if such changes here and below are acceptable here.
WDYT?